src/Routes/Routes.tsx

```diff
@@ -98,7 +98,7 @@ export const Routes = memo(() => {
   // Most routes should be stable and not re-render when the location changes
   // However, some routes are unstable and should actually re-render when location.pathname reference changes
   // Which happens both on actual route change, or when the same route gets pushed
-  const unstableRoutes = useMemo(() => ['/trade'], [])
+  const unstableRoutes = useMemo(() => [], [])
```

Comment:
Ab-solutely, we can't have that one anymore, else this will produce bugs when connecting a wallet (i.e this will re-render the trade route,  and given we use some random key to make the trade route unstable, this will reset the input).

For historical context, the goal was to *actually* reset the swapper when clicking "Trade" in the side nav, for a better UX, but we can't do that anymore.

Reply:
de59ffb1ee3c8562403e2f32e899f311e5dfb1e1